neuraxle/pipeline.py

```diff
@@ -328,7 +328,7 @@ class MiniBatchSequentialPipeline(Pipeline):
     def __init__(self, steps: NamedTupleList, batch_size=None):
         Pipeline.__init__(self, steps)
         self.__validate_barriers_batch_size(batch_size)
-        self.__patch_missing_barrier(batch_size)
+        self._patch_missing_barrier(batch_size)
```

Comment:
not sure why some methods are single underscore and some are double. We should never use double.